components/button/button.tsx

```diff
@@ -157,9 +162,11 @@ export default class Button extends React.Component<ButtonProps, any> {
 
   render() {
     const {
-      type, shape, size, className, htmlType, children, icon, prefixCls, ghost, ...others
+      type, shape, size, className, children, icon, prefixCls, ghost, loading: _loadingProp, ...rest,
```

Comment:
According to built-in tslint.json, it requires trailing comma, but the CI report
```
antd-tools run compile
Without `from` option PostCSS could generate wrong source map and will not find Browserslist config. Set it to CSS file path or to `undefined` to prevent this warning.
components/button/button.tsx(165,101): error TS1013: A rest parameter or binding pattern may not have a trailing comma.
```

Maybe we should confirm the coding style.

Reply:
reinstall antd-tools